src/group_theory/submonoid/pointwise.lean

```diff
@@ -266,7 +304,7 @@ lemma mem_pointwise_smul_iff_inv_smul_mem {a : α} {S : add_submonoid A} {x : A}
   x ∈ a • S ↔ a⁻¹ • x ∈ S :=
 mem_smul_set_iff_inv_smul_mem
 
-lemma mem_smul_pointwise_iff_exists (m : A) (a : α) (S : add_submonoid A) :
+lemma mem_pointwise_smul_iff_exists (m : A) (a : α) (S : add_submonoid A) :
```

Comment:
If you rename these, you should rename the subgroup, subsemiring, subring, and submodule ones too. Probably all these renames belong in a standalone PR

Reply:
Yeah, I'll do the `smul_pointwise` and `subset` renaming on a separate PR another day.